packages/flutter/test/widgets/grid_view_test.dart

```diff
@@ -898,4 +898,61 @@ void main() {
     expect(controller.position.maxScrollExtent, 472.0);
     expect(controller.position.pixels, 472.0);
   });
+
+  testWidgets('SliverGridDelegate mainAxisExtent add assert', (WidgetTester tester) async {
```

Comment:
This tests is currently failing on web, this is why it was reverted previously, Can you take a look?